platform_login/index.js

```diff
@@ -64,7 +64,7 @@ module.exports.handler = (event, context, callback) => {
 			var authenticationDetails = new AWSCognito.AuthenticationDetails(authenticationData);
 			var userPool = new AWSCognito.CognitoUserPool(poolData);
 			var userData = {
-				Username : event.body.username,
+				Username : event.body.username.toLowerCase(),
```

Comment:
I hope this is not a change related to CAPI-1252

Reply:
This is not a change related to CAPI-1252